flow-typed/velocity9.js

```diff
@@ -0,0 +1,26 @@
+declare type UserComponentItem = {
```

Comment:
I'm down to talk about this IRL, idea was originally because it's almost exclusively used in an array. A `UserComponentItem` in my mind doesn't imply at all that it represents the overall component, but `UserComponent` is more ambiguous. Is there perhaps a different postfix we could use?

Reply:
I don’t think the suffix “item” is really ever appropriate.
I use “ComponentId” for this datum in the worker, does that make sense?

Reply:
I don’t think the suffix “item” is really ever appropriate.
I use “ComponentId” for this datum in the worker, does that make sense?